snuba/snuba_migrations/group_attributes/0002_add_priority_to_group_attributes.py

```diff
@@ -0,0 +1,44 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.GROUP_ATTRIBUTES,
+                table_name="group_attributes_local",
+                column=Column("group_priority", UInt(8)),
```

Comment:
@snigdhas can we just call it `priority`? I don't think there is a need to call it `group_priority` when it's on a group table. Also, should there be a default value?

Reply:
sticking with the [existing pattern](https://github.com/getsentry/snuba/blob/92f2ad28d1828e5497f57852e4e2157c8e444e0b/snuba/snuba_migrations/group_attributes/0001_group_attributes.py#L13-L16) where columns from the group table are prefixed with `group_`. 

We _do_ default to Medium priority as the base case on ingestion but I don't believe we need a default here. Since it's a nullable column, it should be fine while we backfilled the rows and then going forwards it should be included in the snapshots.

Reply:
@snigdhas fair enough, let's stick to conventions

Reply:
> Since it's a nullable column

Your migration does not make this column Nullable. You need to add `Modifiers(nullable=True)` to your Column definition.

`UInt(8, Modifiers(nullable=True))`

Reply:
> while we backfilled the rows

How are you planning to do this?

Reply:
using `bulk_send_snapshot_values` in this [migration](https://github.com/getsentry/sentry/pull/69072)